app/assets/stylesheets/components/_intervention.scss

```diff
@@ -0,0 +1,5 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.gem-c-intervention {
+  margin-top: govuk-spacing(4);
```

Comment:
Is this css styling going to work for all page types that might have a banner? As they all have different templates, did you/could you check and add screenshots to the PR description?